src/js/components/campaignpage/CampaignSplashCard.jsx

```diff
@@ -33,15 +33,15 @@ const CampaignSplashCard = ({ branding, shareMeta, pdf, campaignPage, donationVa
 
 	const donationDisplay = <b>{donationValue ? <Counter amount={donationValue} minimumFractionDigits={2} preserveSize /> : "money"}</b>;
 
+	let [x, setX] = useState(undefined)
+	if(!x) {donationValue.value = 1340236.12; setX(donationValue);}
+
 	let splashText = <>
-		<div className="header text-white">
-			<span>
-				{ongoing ? "Raising " : "Raised "}
-				{donationDisplay} for
-				{' '}{charityName}
-			</span>
+		<div className="text-uppercase text-white splash-thanks">
+			Thank you for watching {nvertiserName}'s advert and donating!
 		</div>
-		<p className="text-white subtext">by using purpose led online ads</p>
+		<p className="text-white raised splash-counter">{<Counter amount={x} minimumFractionDigits={2} preserveSize /> }</p>
+		<p className='text-white splash-gl'>Raised for charity by using purpose lead online ads with Good-Loop</p>
```

Comment:
@LewisFramps Hm: the previous code had logic for present vs past tense. That's probably still wanted (and probably best to check the rest of the page's text for any tense issues). Maybe chat with Carol or Fiona.

Reply:
messaged Fiona, got an update ready to push if we're wanting to keep it